### DISCUSSION TITLE
Please upgrade alpine image jdk version

### DISCUSSION THREAD

**Discussion by fsdrw08** [OpenIdentityPlatform/OpenDJ#337] [Ideas]
Currently the [alpine docker image](https://github.com/OpenIdentityPlatform/OpenDJ/blob/master/opendj-packages/opendj-docker/Dockerfile-alpine#L1) still using jdk8, please consider upgrading to jdk19 or 21 just like the debian image

**Comment by vharseko** [comment]
please check https://github.com/OpenIdentityPlatform/OpenDJ/issues/328
